src/Files.App/UserControls/Widgets/FileTagsWidget.xaml

```diff
@@ -51,39 +52,44 @@
 					</Grid.RowDefinitions>
 
 					<!--  Title  -->
-					<Grid BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}" BorderThickness="0,0,0,1">
+					<Grid
+						Background="Transparent"
+						BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+						BorderThickness="0,0,0,1">
 						<Grid.ColumnDefinitions>
 							<ColumnDefinition Width="Auto" />
 							<ColumnDefinition Width="*" />
 							<ColumnDefinition Width="Auto" />
 						</Grid.ColumnDefinitions>
 
-						<StackPanel
-							Padding="12,8,12,8"
-							Orientation="Horizontal"
-							Spacing="8">
-							<!--  Tag Color  -->
-							<PathIcon Data="{StaticResource ColorIconFilledTag}" Foreground="{x:Bind Color, Mode=OneWay, Converter={StaticResource StringToBrushConverter}}" />
-
-							<!--  Tag Name  -->
-							<TextBlock
-								Margin="0,-2,0,0"
-								FontWeight="SemiBold"
-								Text="{x:Bind Name, Mode=OneWay}" />
-						</StackPanel>
-
 						<!--  View More  -->
 						<HyperlinkButton
-							Grid.Column="2"
 							Margin="4"
 							AutomationProperties.Name="{helpers:ResourceString Name=ViewMore}"
 							Command="{x:Bind ViewMoreCommand}"
 							ToolTipService.ToolTip="{helpers:ResourceString Name=ViewMore}">
-							<HyperlinkButton.Content>
-								<FontIcon
-									FontSize="12"
+							<StackPanel Orientation="Horizontal" Spacing="8">
+								<!--  Tag Color  -->
+								<PathIcon Data="{StaticResource ColorIconFilledTag}" Foreground="{x:Bind Color, Mode=OneWay, Converter={StaticResource StringToBrushConverter}}" />
+
+								<!--  Tag Name  -->
+								<TextBlock
+									Margin="0,-2,0,0"
 									FontWeight="SemiBold"
-									Glyph="&#xE76C;" />
+									Foreground="{ThemeResource TextFillColorPrimaryBrush}"
+									Text="{x:Bind Name, Mode=OneWay}" />
+							</StackPanel>
+						</HyperlinkButton>
+
+						<!--  Open All Items  -->
+						<HyperlinkButton
+							Grid.Column="2"
+							Margin="4"
+							AutomationProperties.Name="{helpers:ResourceString Name=OpenAllItems}"
+							Command="{x:Bind OpenAllCommand}"
+							ToolTipService.ToolTip="{helpers:ResourceString Name=OpenAllItems}">
```

Comment:
If the resources strings are indeed renamed as proposed, these entries will need to be renamed as well.